core/src/main/java/com/datastax/oss/driver/internal/core/context/DefaultDriverContext.java

```diff
@@ -563,9 +562,12 @@ protected RequestProcessorRegistry buildRequestProcessorRegistry() {
     return new RequestProcessorRegistry(logPrefix, processors.toArray(new RequestProcessor[0]));
   }
 
-  protected CodecRegistry buildCodecRegistry(String logPrefix, List<TypeCodec<?>> codecs) {
-    MutableCodecRegistry registry = new DefaultCodecRegistry(logPrefix);
-    registry.register(codecs);
+  protected CodecRegistry getCodecRegistry(ProgrammaticArguments arguments) {
```

Comment:
Why change the method name? This is not a huge deal, but the rest of the class follows the convention `buildXxx` for the method that creates the object (either at construction time or when the lazy reference gets accessed), and `getXxx` for the public method that clients call to get the component. We already have a no-arg `getCodecRegistry` for the latter.

Reply:
Fully agreed, undid the rename.